plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoClientConfig.java

```diff
@@ -59,6 +64,38 @@
     private String requiredReplicaSetName;
     private String implicitRowFieldPrefix = "_pos";
 
+    @AssertTrue(message = "Exactly one of 'mongodb.schema-database' or 'mongodb.schema-collection' must be specified, or the default _schema could be created in the same database")
+    public boolean isSchemaManagementConfigValid()
+    {
+        return schemaDatabase.isEmpty() || schemaCollection.equals("_schema");
+    }
+
+    public boolean getEnableCreateIndex()
+    {
+        return enableCreateIndex;
+    }
+
+    @Config("mongodb.enable-schema-create-index")
+    public MongoClientConfig setEnableCreateIndex(boolean enableCreateIndex)
+    {
+        this.enableCreateIndex = enableCreateIndex;
+        return this;
+    }
+
+    @NotNull
+    public Optional<String> getSchemaDatabase()
+    {
+        return schemaDatabase;
+    }
+
+    @ConfigDescription("Database to use for managing schemas of collections in the different databases.")
+    @Config("mongodb.schema-database")
+    public MongoClientConfig setSchemaDatabase(String schemaDatabase)
+    {
+        this.schemaDatabase = Optional.of(schemaDatabase);
```

Comment:
Change to `Optional.ofNullable` and then we can simplify config tests.

Reply:
Sure